en_US/changes/changes.md

```diff
@@ -15,6 +15,40 @@ ref: undefined
 
 # Changes
 
+## Version 4.3-alpha.1
+
+*Release Date: 2020-01-29*
+
+EMQ X 4.3-alpha.1 is released now, it mainly includes the following changes:
+
+*Features*
+
+- Support Erlang/OTP 23
+- The new installation package only supports macOS 10.14 and above
```

Comment:
this does not look like a featrue, maybe create a 'others' section for it.

Reply:
Maybe we shouldn't discuss these in this PR, I will merge it first, and then we will look for another opportunity to discuss how to improve it.